docs/intro/serverless.md

```diff
@@ -49,7 +49,7 @@ To create GraphQL Layer at runtime developers need to initialize `GraphbackRunti
   });
 ```
 
-See [`./runtime.ts`](https://github.com/aerogear/graphback/blob/master/templates/ts-apollo-runtime-backend/src/runtime.ts) for a fully functional example.
+See our [`ts-apollo-runtime-backend`](https://github.com/aerogear/graphback/blob/master/templates/ts-apollo-runtime-backend/src/runtime.ts) for a fully functional example.
 
 ### GraphbackDataProvider
```

Comment:
This completely outdated. Are we going to do it later?

Reply:
Yeah I didn't really get to modify this page.